specification/dimensions/availabilityzone.md

```diff
@@ -0,0 +1,30 @@
+# Availability Zone
+
+Availability zones are physically separated and isolated areas within a region which provide high availability and fault tolerance.
+
+The AvailabilityZone column MUST be present in the billing data. This column MUST be of type String and MAY contain null values.
+
+## Column ID
+
+AvailabilityZone
+
+## Display name
+
+Availability Zone
+
+## Description
+
+Availability zones are physically separated and isolated areas within a region which provide high availability and fault tolerance.
```

Comment:
```suggestion
Physically separated and isolated areas within a region which provide high availability and fault tolerance.
```

Reply:
It looks like this change was performed on the row 3 (within Definition) instead of the row 17 (within Description)